File: app/src/main/java/bin/mg/main/core/info/ApkInfo.java
package bin.mg.main.core.info;

import android.graphics.drawable.Drawable;
import bin.mg.main.model.FileItem;

public class ApkInfo {
    
    String packageName;
    String versionName;
     long versionCode;
    long fileSize;
    boolean isInstalled;
    boolean isSystemApp;
    boolean signatureV1;
    boolean signatureV2;
    boolean signatureV3;
    int targetSdk;
    int minSdk;
    String installedVersion;
    String dataDir;
    String sourceDir;
    long firstInstallTime;
    long lastUpdateTime;
    int uid;
    Drawable appIcon;
    String appName;
    
    public void getPackageName(FileItem item) {
        
    }
}

File: app/src/main/java/bin/mg/main/app/ApkInfo.kt
package bin.mg.main.app

import android.graphics.drawable.Drawable
import bin.mg.main.model.FileItem

class ApkInfo {
    var packageName: String? = null
    var versionName: String? = null
    var versionCode: Long = 0
    var fileSize: Long = 0
    var isInstalled: Boolean = false
    var isSystemApp: Boolean = false
    var signatureV1: Boolean = false
    var signatureV2: Boolean = false
    var signatureV3: Boolean = false
    var targetSdk: Int = 0
    var minSdk: Int = 0
    var installedVersion: String? = null
    var dataDir: String? = null
    var sourceDir: String? = null
    var firstInstallTime: Long = 0
    var lastUpdateTime: Long = 0
    var uid: Int = 0
    var appIcon: Drawable? = null
    var appName: String? = null

    fun getPackageName(item: FileItem) {
        // Stub
    }
}